rpcs3/Emu/RSX/Common/FragmentProgramDecompiler.cpp

```diff
@@ -520,27 +521,81 @@ template<typename T> std::string FragmentProgramDecompiler::GetSRC(T src)
 			"ssa"
 		};
 
-		//TODO: Investigate effect of input modifier on this type
+		// NOTE: Hw testing showed the following:
+		// 1. Reading from registers 1 and 2 (COL0 and COL1) is clamped to (0, 1)
+		// 2. Reading from registers 4-12 (inclusive) is not clamped, but..
+		// 3. If the texcoord control mask is enabled, the last 2 values are always 0 and 1!
+		// 4. Reading from tc9 returns one value in w (point size?) and contains a 1
```

Comment:
Could this be perspective shear rather than point size?

Reply:
Comment is outdated, turns out I had the enable bit for the register in the wrong place. Properly initializing it shows that it does indeed carry information like the other texcoord registers.